lineman/app/components/dashboard_page/dashboard_page_controller.coffee

```diff
@@ -1,97 +1,60 @@
-angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, Records, CurrentUser, LoadingService) ->
+angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, Records, CurrentUser, LoadingService, ThreadQueryService) ->
   $rootScope.$broadcast('currentComponent', 'dashboardPage')
   $rootScope.$broadcast('setTitle', 'Dashboard')
 
-  @loaded =
-    sort_by_date:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
-    sort_by_group:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
-  @perPage =
-    sort_by_date: 25
-    sort_by_group: 10
-  @groupThreadCounts =
-    hidden:    0
-    collapsed: 5
-    expanded:  10
-
-  @sort   = -> CurrentUser.dashboardSort
-  @filter = -> CurrentUser.dashboardFilter
-
-  @loadParams = ->
-    filter: @filter()
-    per:    @perPage[@sort()]
-    from:   @loadedCount()
-
-  @loadedCount = (group) =>
-    if group
-      @groupThreadCounts[group.dashboardStatus or 'collapsed']
-    else
-      @loaded[@sort()][@filter()]
+  Records.votes.fetchMyRecentVotes()
 
-  @loadMore = (options = {}) =>
-    @loaded[@sort()][@filter()] = @loadedCount() + @perPage[@sort()]
-    switch @sort()
-      when 'sort_by_date'  then Records.discussions.fetchInboxByDate(@loadParams())
-      when 'sort_by_group' then Records.discussions.fetchInboxByGroup(@loadParams())
+  @perPage = 25
+  @loaded =
+    show_all:           0
+    show_muted:         0
+    show_proposals:     0
+    show_participating: 0
+  @filter    = -> CurrentUser.dashboardFilter
+  @nowLoaded = -> @loaded[@filter()]
+
+  @timeframes =
```

Comment:
The dashboard can handle two different types of views for any filter, one for timeframes and one for groups.

I didn't implement the current / past proposals thing yet... it strikes me as likely to get designed away soon, heh. >.>